extensions/open-dir-in-finder/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "open-dir-in-finder",
+  "title": "Open in Finder",
```

Can we change this title to something more descriptive? It's what shows up in the store, so people should understand what they install...

```suggestion
  "title": "Open in Finder",
```